.travis.yml

```diff
@@ -10,27 +10,36 @@ env:
   - USER_TABLE: testing
   - secure: ldfVKFlO/kQLAmXFTf+UIsj1OzGV+EtfF1mEqp9ArD9eZg0EiM6mr2LTWMAKUyUlD3uCXSkF0xmIUIxNSUB+FwDxSNZdtO+Fx3hmF6I6ofX2NGEDdz2l/F7LeC3MfyPU7qwL9IL15gX2w6got78eDtdGMlhmt2s8dNsWDbLlAMjGZtCvfcXjY5GJS96Pe4lNHbCNlAi+x8E2UsyOBAEAySXfCfbS1JLW9XqiBKk1hinE+f2j47Wb6DJVX3JKfYLgdRdO+4JVa5dJvBIGhmO3lTi+vwDpTx+QbI+/oaoLpqSBpPwcpycZSPCJ0Oo+HfR8DCQRmT8NB6uc/wD44hrYlOXfXudaUH1TzQ2eT+ZT6BY5ikCArHpRDFHzlu2XmbTRq+phzArlV54H8xd46sIrMkLlVP4UHQ1GVOnXWTlG6wkrXxyPUFPAFeSKOiiK2UoHgz6j7yOTwKkuYH3/96GzH+YmoNZPFaDCh31YDFW4+ynlHFWA96U9+GKVS66f52aNMf7El6SwYSeuOzfOnKo5h5opKkkjO83KVrfzwBwpYkXHtyMG2o8hhJUDBF6VozPusJllkZfdyapdQgdTtH5R4Es1QNkzLAfX/iNGfr1yMc+I6O7VLEoB30202m1GDaocqnUwCHnMhnPdkw8E1x0DXoqSBHlCr/GJMjNiRsPjZ0k=
 before_install:
-# https://github.com/travis-ci/travis-ci/issues/8982
-#- python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
-# https://github.com/travis-ci/travis-ci/issues/4704
-- python -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); fcntl.fcntl(sys.stdout, fcntl.F_SETFL, flags&~os.O_NONBLOCK);'
-- python -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); print(flags&os.O_NONBLOCK);'
-- sudo rm -f /etc/boto.cfg
-- pip install -r automation_requirements.txt
-- npm install -g dynalite
-- npm install -g kinesalite
+  - sudo apt-get update
+  - sudo apt-get install -y golang
+  - go get -u github.com/stripe/stripe-mock
+  - sudo echo api.stripe.com > localhost
+  # https://github.com/travis-ci/travis-ci/issues/8982
+  #- python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
+  # https://github.com/travis-ci/travis-ci/issues/4704
+  - python -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); fcntl.fcntl(sys.stdout, fcntl.F_SETFL, flags&~os.O_NONBLOCK);'
+  - python -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); print(flags&os.O_NONBLOCK);'
+  - sudo rm -f /etc/boto.cfg
+  - npm install -g dynalite@2.3.2
+  - npm install -g kinesalite@2.0.3
 script:
-- mkdir -p ~/.aws/
-- |
-  cat >~/.aws/credentials <<-EOF
-  [default]
-  aws_access_key_id=fake-id
-  aws_secret_access_key=fake-key
-  EOF
-- |
-  cat >~/.aws/config <<-EOF
-  [default]
-  output=json
-  region=us-west-2
-  EOF
-- doit test
+  - mkdir -p ~/.aws/
+  - |
+    cat >~/.aws/credentials <<-EOF
+    [default]
+    aws_access_key_id=fake-id
+    aws_secret_access_key=fake-key
+    EOF
+  - |
+    cat >~/.aws/config <<-EOF
+    [default]
+    output=json
+    region=us-west-2
+    EOF
+  - pip install -r automation_requirements.txt
+  - |
+    echo "starting stripe-mock"
+    stripe-mock -http-port 80 -https-port 443 &
+    STRIPE_MOCK_PID=$!
```

Comment:
This line could be removed as I am letting the operating system collect this handle once the job is complete.